### PR TITLE
SR-IOV changes 

### PR DESCRIPTION
**Pull request by SanaZulfiqar**
- Fixed names of sriov network environment files from "sriov_7_ports"/"sriov_9_ports" to "sriov_7_port"/"sriov_9_port" to follow the same naming pattern.
- Fixed typo in the comment.
- changed default value of "sriov_vf_count" parameter.

### CODE REVIEW

**Comment by gaelrehault** [comment]
+1

File: src/deploy/osp_deployer/settings/sample_csp_profile.ini [hunk 1]
```diff
@@ -167,6 +167,10 @@ sah_bond_opts=mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
 #       where X represents the total number of ports:
 #            9: Five + Four DPDK ports
 #            7: Five + Two DPDK ports
+#       If SR-IOV needs to enable, then use
```
Comment (gaelrehault) [inline src/deploy/osp_deployer/settings/sample_csp_profile.ini]: Should read " If SR-IOV is enabled, then.."

File: src/deploy/osp_deployer/settings/sample_xsp_profile.ini [hunk 2]
```diff
@@ -167,6 +167,10 @@ sah_bond_opts=mode=802.3ad miimon=100 xmit_hash_policy=layer3+4 lacp_rate=1
 #       where X represents the total number of ports:
 #            9: Five + Four DPDK ports
 #            7: Five + Two DPDK ports
+#       If SR-IOV needs to enable, then use
```
Comment (gaelrehault) [inline src/deploy/osp_deployer/settings/sample_xsp_profile.ini]: Should read " If SR-IOV is enabled, then.."

**Comment by heydog15** [comment]
Looks good.

File: src/deploy/osp_deployer/settings/sample_xsp_profile.ini [hunk 3]
```diff
@@ -201,28 +200,24 @@ StorageBond1Interface1=em2
...
 #ComputeOvsDpdkInterface2=p3p1
 #ComputeOvsDpdkInterface3=p2p2
 #ComputeOvsDpdkInterface4=p3p2
 #BondInterfaceOvsOptions=bond_mode=balance-tcp lacp=active
 
-# Note: the following interfaces need to be changed as per the server model
-# To enable standalone SR-IOV, two or four interfaces should be used
-# For enabling SR-IOV with OVS-DPDK, two interfaces should be used.
+# To enable standalone SR-IOV, two or four interfaces should be used.
+# To enable SR-IOV with OVS-DPDK, two interfaces should be used.
```
Comment (heydog15) [inline src/deploy/osp_deployer/settings/sample_xsp_profile.ini]: Shouldn't four interfaces be needed when both SR-IOV and OVS-DPDK are used?

**Pull request merged**